src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -37,6 +37,8 @@
 */
 public without sharing class UTIL_CustomSettingsFacade {
 
+    public Boolean hasPerm {get;set;}
```

Comment:
Does this need to be public for the component to be able to read it? If not, better make it private.